Formula/elasticsearch.rb

```diff
@@ -34,6 +35,10 @@ def install
     # Install everything else into package directory
     libexec.install "bin", "config", "lib", "modules"
 
+    inreplace "#{libexec}/bin/elasticsearch-env",
+              "if [ -z \"$ES_PATH_CONF\" ]; then ES_PATH_CONF=\"$ES_HOME\"/config; fi",
```

Comment:
Just the `then ES_PATH_CONF=\"$ES_HOME\"/config; fi",` should be enough

Reply:
I prefer it this way, it's clearer to think about to simply replace the whole line.

Reply:
It goes passed the eightieth column, which is :-1:

Reply:
That's not what the audit rule enforces. I don't think different rules should be enforced in PR reviews than are enforced by audit. I don't think less clear code should be written to fit in 80 columns if that's not what is enforced in audit.

Reply:
It's not enforced in audit because sometimes it is unavoidable, and I don't want to deal with red CI due to things we cannot avoid. This one is avoidable.